CredentialsContainer.js

```diff
@@ -15,7 +15,14 @@ const CREDENTIAL_STORE_TIMEOUT = 0;
 
 export class CredentialsContainer {
   constructor(injector) {
-    this._nativeCredentialsContainer = navigator.credentials;
+    if (typeof navigator.credentials === "object") {
+      this._nativeCredentialsContainer = {};
+      this._nativeCredentialsContainer.get = navigator.credentials.get.bind(navigator.credentials);
+      this._nativeCredentialsContainer.store = navigator.credentials.store.bind(navigator.credentials);
+      this._nativeCredentialsContainer.create = navigator.credentials.create.bind(navigator.credentials);
+      this._nativeCredentialsContainer.preventSilentAccess = navigator.credentials.preventSilentAccess.bind(navigator.credentials);
```

Comment:
Maybe we just use a loop to bind all functions that appear on the object. Thoughts?

Reply:
Something like:

``` js
Object.keys(navigator.credentials).forEach((key) => {
  if(typeof navigator.credentials[key] === 'function')
    this._nativeCredentialsContainer[key] = navigator.credentials[key].bind(navigator.credentials)];
});
```

Same number of lines of code, but more obfuscated... Also doesn't help unless you somehow attach them to `CredentialsContainer` too.

Reply:
`Object.keys` wouldn't work.  Would need a `for(const key in navigator.credentials)` sort of construct I think.

Reply:
Another option:
```js
['get', 'store', 'create', 'preventSilentAccess'].forEach(key =>
  this._nativeCredentialsContainer[key] = navigator.credentials[key].bind(navigator.credentials)
);
```

Reply:
I just realized that we need to use a dynamic loop since various implementations (e.g. Firefox) may not have all the same methods implemented. That loop should assign existing methods to the `CredentialsContainer` instance, but catch `get` and `store`. Also, this would obviate the current `CredentialsContainer#create` and `CredentialsContainer#preventSilentAccess` methods.

I'll update my PR accordingly tonight.